lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -483,13 +484,13 @@ export class UserAgentApplication {
      * Helper function to acquireToken
      *
      */
-    private acquireTokenHelper(account: Account, interactionType: InteractionType, isLoginCall: boolean, request: AuthenticationParameters, resolve?: any, reject?: any): void {
+    private async acquireTokenHelper(account: Account, interactionType: InteractionType, isLoginCall: boolean, request: AuthenticationParameters, resolve?: any, reject?: any): Promise<void> {
```

Comment:
Does this effect performance for calls that don't need this async?

Reply:
Potentially, but should be offset when not making the openid-config call at all. Will verify.